app/controllers/users_controller.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[edit update destroy]
+
+  # GET /users or /users.json
+  def index
+    @users = User.all
+  end
+
+  # GET /users/1/edit
+  def edit; end
+
+  # PATCH/PUT /users/1 or /users/1.json
+  def update
+    respond_to do |format|
+      if @user.update(user_params)
+        format.html { redirect_to users_url, notice: "User was successfully updated." }
+        format.json { render :index, status: :ok, location: @user }
```

Comment:
I believe your json responses aren't needed just now. (Use it in case you are doing an AJAX query through JS or something that explicitly requires for a json response 😁)